utils/image/imageZoom.ts

```diff
@@ -0,0 +1,46 @@
+import { BLOG_BUCKET_URL } from '../../store/apis';
+
+const GOOGLE_IMG_API = 'googleusercontent.com';
+const GITHUB_IMG_API = 'avatars.githubusercontent.com';
+const TELEGRAM_IMG_API = 't.me';
+
+export type TImageSources = 'google' | 'github' | 'upper' | 'telegram' | 'undefined';
```

Comment:
there will not be images from other sources, another source is not configured for cors in next.config.js. You should remove undefined type

Reply:
@samandar-boymurodov you did not resolve this